### PR TITLE
added Will and Sophie

### PR DESCRIPTION
**Pull request by mbobra**
I added Will Barnes as the Deputy Lead Developer and Sophie Murray as the Communication and Education Lead.

### CODE REVIEW

File: team.rst [hunk 1]
```diff
@@ -109,23 +109,23 @@ Community Roles
...
     :desc: Stuart is the Python developer for the Daniel K. Inouye Solar Telescope Data Centre. He obtained a PhD in Numerical solar physics from Sheffield University in 2016, prior to his PhD he obtained a first class MPhys degree in Physics with Planetary and Space Physics from The University of Wales Aberystwyth, during which he spent 5 months studying at UNIS on Svalbard in the high arctic.
 
-.. card:: Nabil Freij
-    :img_name: nabil.png
-    :github: nabobalis
-    :title: Deputy Lead Developer and Release Manager
-    :aff_name: Freelance
-    :aff_link:
-    :date: 17 March 2016
-    :desc: Nabil Freij is working as  Freelance Python developer. He was a research scientist at Universitat de les Illes Balears working on coronal heating and MHD waves. Nabil received a Master of Physics at the University of Warwick and a Ph.D. in Applied Mathematics from the University of Sheffield.
```
Comment (nabobalis) [inline team.rst]: I was hoping to remove myself.

Comment (mbobra) [reply team.rst]: Sorry, totally. Do you want me to remove this PR?

Comment (mbobra) [reply team.rst]: We really appreciate you Nabil! 😭

Comment (mbobra) [reply team.rst]: I closed the PR. We should have talked first, it is my bad. I didn't mean to offend anyone. I apologize!

Comment (nabobalis) [reply team.rst]: I was joking @mbobra, I am sorry I have caused this confusion.

Comment (nabobalis) [reply team.rst]: You also deleted the repository (github allows you to do that from a PR for some reason)?

**Pull request closed without merging**